xml/System.Reflection/MethodImplAttributes.xml

```diff
@@ -681,7 +683,7 @@
       </ReturnValue>
       <MemberValue>1024</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Specifies that the JIT compiler should look for security mitigation attributes, such as the user-defined <see cref="System.Runtime.CompilerServices.SecurityMitigationsAttribute" />. If found, the JIT compiler applies any related security mitigations. Available starting with .NET Framework 4.8.</summary>
```

Comment:
This API doesn't seem to exist @rpetrusha and the right link would start with T: